project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { cuisine: "" }
+    this.changeState = this.changeState.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  // Updates the state of the input element so it holds the chosen value.
+  changeState(event) {
+    this.setState({ cuisine: event.target.value })
```

Comment:
Fun trick here - you can just do `this.setState({ [event.target.name]: event.target.value })`. 
`event.target.name` will get the value of the `name` prop from the input (i.e. `name="cuisine"`). Just make sure you're naming the name props on your form inputs the same as you are naming your fields in state.
Once you get more inputs in your state this will be helpful, so you don't have to write out each one!

Reply:
How you have it is good for now because we only have cuisine in state, but in a follow up PR when you add another field to state, I would do this so you dont have to write multiple handleChange methods :)